src/utils/templates/ListTemplate.js

```diff
@@ -0,0 +1,39 @@
+import { Button } from '@mui/material';
+import './style.css'
+import { useNavigate } from 'react-router-dom';
+
+const ListTemplate = ({ templateData, templateType }) => {
+    const templateCode = templateType.charCodeAt(0)
```

Comment:
Would be good to check if `templateType` isn't null/or an empty string before using it